src/main/java/org/stellar/sdk/BumpFootprintExpirationOperation.java

```diff
@@ -0,0 +1,71 @@
+package org.stellar.sdk;
+
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.experimental.SuperBuilder;
+import org.stellar.sdk.xdr.BumpFootprintExpirationOp;
+import org.stellar.sdk.xdr.ExtensionPoint;
+import org.stellar.sdk.xdr.OperationType;
+import org.stellar.sdk.xdr.Uint32;
+
+/**
+ * Represents <a
+ * href="https://developers.stellar.org/docs/fundamentals-and-concepts/list-of-operations#bump-footprint-expiration"
```

Comment:
this docs link isn't valid yet, the public docs haven't been updated for the new soroban ops yet. Can you update these references to point at the go struct defs for the json model instead, it's located in the `soroban-xdr-next` branch of [stellar/go/protocols/horizon/main](https://github.com/stellar/go/blob/soroban-xdr-next/protocols/horizon/operations/main.go#L376-L381)

Reply:
Fixed in 3eb0d1a63707653df7ee111b3682f729127c6197, I also updated the links in the code related to InvokeHostFunction.